tst/com/amazon/corretto/crypto/provider/test/TestRunner.java

```diff
@@ -116,8 +117,8 @@ public static void main(String[] args) throws InitializationError {
         } else {
             printSystemInfo();
             final String suiteName = args[1];
-            final Class[] parallel_classes = SUITES_PARALLEL.get(suiteName);
-            final Class[] serial_classes = SUITES_SERIAL.get(suiteName);
+            Class[] parallel_classes = SUITES_PARALLEL.get(suiteName);
+            Class[] serial_classes = SUITES_SERIAL.get(suiteName);
```

Comment:
These might be easier to manipulate if you replace them with ```List<Class> parallel_classes = new ArrayList<Class>(Optional.ofNullable(SUITS_PARALLEL).orElse(new Class[0]))``` and then do normal List manipulation below? But entirely a style thing.